pkg/features/kube_features.go

```diff
@@ -163,6 +163,13 @@ const (
 	//
 	// Enable admission fair sharing
 	AdmissionFairSharing featuregate.Feature = "AdmissionFairSharing"
+
+	// owner: @mwysokin @mykysha @mbobrovskyi
+	// TODO: Update after merge
```

Comment:
Update what exactly?

Reply:
KEP is already merged https://github.com/kubernetes-sigs/kueue/tree/main/keps/1618-optional-gc-of-workloads. So we can change it now.